src/hotspot/os_cpu/linux_arm/safefetch_linux_arm.S

```diff
@@ -26,6 +26,7 @@
     .globl SafeFetch32_impl
     .globl _SafeFetch32_fault
     .globl _SafeFetch32_continuation
+    .type SafeFetch32_impl, %function
```

Comment:
By adding this .type directive, the compiler knows that SafeFetch32_impl is a function. 
When static linker resolves it, it will update the correct branch instruction according to its target. In this case, it will use BX on Ubuntu18.04. 

Is my understanding correct?

Reply:
> By adding this .type directive, the compiler knows that SafeFetch32_impl is a function. When static linker resolves it, it will update the correct branch instruction according to its target. In this case, it will use BX on Ubuntu18.04.
> 
> Is my understanding correct?

Correct.